src/System.DirectoryServices/src/System.DirectoryServices.csproj

```diff
@@ -160,10 +162,14 @@
     <Reference Include="System.Net.NameResolution" />
     <Reference Include="System.Net.Primitives" />
     <Reference Include="System.Net.Security" />
+    <Reference Include="System.Runtime" />
     <Reference Include="System.Resources.ResourceManager" />
     <Reference Include="System.Security.Claims" />
     <Reference Include="System.Security.Cryptography.Algorithms" />
     <Reference Include="System.Threading.Thread" />
     <Reference Include="System.Threading" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp2.0'">
```

Comment:
Is this condition valid for all the relevant configs? What about the netstandard configuration?

Reply:
Someone more knowledgeable than me would have to comment on this.

Reply:
@joperezr - can you confirm that this change is valid?

Reply:
Yes this LGTM. We build this project for three configurations: netstandard, netcoreapp 2.0, and netcoreapp 3.0. The only one that needs the additional System.Memory reference is netcoreapp2.0, as netcoreapp3.0 and netstandard configurations would not need System.Memory as it should just be a facade for those configs.